### DISCUSSION TITLE
Mass suppression of PossiblyInvalidArgument with a specific pattern?

### DISCUSSION THREAD

**Discussion by Saeven** [vimeo/psalm#10865] [Q&A]
Hey everyone.  I'm using Laminas, and it seems there's a bit of an issue with returns and the container interface.  It looks a little like this:

https://psalm.dev/r/b39c19e956

Would there be a way for me to suppress all PossiblyInvalidArgument for `->setInputFilter` on the Form parent class?

With thanks!
Alex

**Comment by edsrzf** [comment]
I think It should be possible to suppress this [through the configuration](https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#config-suppression). Something like: (untested)

```xml
<PossiblyInvalidArgument>
  <errorLevel type="suppress">
    <referencedMethod name="Form::setInputFilter" />
  </errorLevel>
</PossiblyInvalidArgument>
```

Alternatively you might be able to do something with a stub file to override either the return type or the parameter type.
